backend/src/main/java/com/redhat/cloud/notifications/routers/models/behaviorgroup/CreateBehaviorGroupRequest.java

```diff
@@ -35,4 +50,13 @@ public class CreateBehaviorGroupRequest {
      * the behavior group).
      */
     public Set<UUID> eventTypeIds;
+
+    /**
+     * Validates that the bundle is identifiable.
+     * @return true if the bundle ID is null and the bundle name is blank. False otherwise.
+     */
+    @AssertFalse(message = "either the bundle name or the bundle UUID are required")
+    private boolean isBundleUuidNullOrBundleNameBlank() {
+        return this.bundleId == null && StringUtils.isBlank(this.bundleName);
```

Comment:
`StringUtils.noBlank` may not be available anymore with the latest version of `master`. `org.apache.commons.lang3.StringUtils` came from a transitive dependency which may no longer be in the classpath. If so, you can simply do an old-school check 😄

Reply:
It's already used elsewhere, if master is not failing, then it should be fine.
I wonder if we should depend on it directly - unsure about maven semantics i.e. it's posible to tell maven to depend on the lib on at least version x.y.z ?

Reply:
It seems like we could use the [`maven-enforcer-plugin`](https://maven.apache.org/enforcer/maven-enforcer-plugin/) to set the minimum version of `commons-lang` to `3.12.0`, which is the one we are currently using? Otherwise I don't mind doing it the old way.

Reference: https://stackoverflow.com/questions/32971660/enforce-a-minimum-version-of-maven

Reply:
All right I just saw that the pipeline is failing because of the missing dependency

Reply:
That specific dependency is declared in the [Quarkus BOM](https://github.com/quarkusio/quarkus/blob/main/bom/application/pom.xml) so if we add it to our POM, we should use the version from Quarkus (which means we won't declare any version as it will come from the BOM).

Reply:
However, do we really need to add a direct dependency only to avoid writing an `if (x != null && !x.isBlank())` condition? This feels like we'll use a bazooka to kill a fly to me 😄

Reply:
All right, I've removed the `StringUtils` bit :+1: